documentation/install.rst

```diff
@@ -72,44 +72,46 @@ The basic dependencies for this type of installanion are very similar to these d
 - clang  with development packages (headers)
 
 The installation process of the required packages varies from system to system.
-Some examples are described in detail below.
-
-
-- For the CentOS7/RHEL7 and compatible systems the llvm-toolset-7.0 toolset from
-  https://www.softwarecollections.org/en/scls/rhscl/llvm-toolset-7.0/ should be installed. Run as root
-
- ``yum install llvm-toolset-7.0* libcxx libcxx-devel``
-
- Then the compilation can be performed using the following shell
-
- ``scl enable llvm-toolset-7.0 bash``
-
-
-- For the Fedora 31+ the dependencies can be installed from the standard repositories. Run as root 
+On the RHEL7/RHEL8/Fedora22+/Ubuntu18+  systems binder can be compilled with the llvm, clang and dependent packages available 
+for these systems from their default repositories
  
-  ``yum install clang clang-devel llvm-devel llvm-static clang-libs libcxx libcxx-devel``
-
-
-- For the CentOS8/RHEL8 the dependencies are the same, but the ``libcxx`` 
-  and ``libcxx-devel`` packages are not available yet from standard repositories
-  so one has to recompile them, i.e.
-
-  ``rpm -i http://ftp.tu-chemnitz.de/pub/linux/fedora/linux/releases/31``
-  ``/Everything/source/tree/Packages/l/libcxx-9.0.0-1.fc31.src.rpm``
-
-  ``wget https://src.fedoraproject.org/rpms/libcxx/raw/f31/f/libcxx.spec``
-
-  edit ``libcxx.spec`` to set
+ 
+For RHEL7/RHEL8/Fedora22+:
 
-  ``%global bootstrap 1``
+- To install the needed packages   run as root 
+ 
+  ``yum install clang clang-devel llvm-devel llvm-static clang-libs``
   
-  Then run as user
-
-  ``rpmbuild -bb libcxx.spec``
+- If a newer or specific version of the llvm/clang is needed, it can be installed  as root
   
-  And run as root
+  ``yum install clang8.0 clang8.0-devel llvm8.0-devel llvm8.0-static clang8.0-libs``
+   
+  to obtain a specific version (8.0 in this case).
+    
+- If the option above is not sufficient, or the available packages are outdated, for the 
+  CentOS/RHEL/Fedora and compatible systems the llvm-toolset-7.0 toolset (or later) from
+  https://www.softwarecollections.org/en/scls/rhscl/llvm-toolset-7.0/ provides . Run as root
```

Comment:
One can any of them. That is the point.  A typical Fedora user would not use software collection just because Fedora contains the newest versions of everything. On the THEL one could easily use the default but old  version, but if a more recent version of llvm would be needed, it should be taken from collections.